api4/webhook.go

```diff
@@ -335,6 +335,9 @@ func createOutgoingHook(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	hook.SecretToken = model.NewRandomString(26)
```

Comment:
This feels like it doesn't belong in `api4`, but in `c.App.CreateOutgoingWebhook(hook)`.